packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -26,8 +25,11 @@ import { HomePath } from '../../constants';
 import { observeNavigationTiming } from 'fxa-shared/metrics/navigation-timing';
 import sentryMetrics from 'fxa-shared/lib/sentry';
 import PageAvatar from './PageAvatar';
+import { QueryParams } from '../..';
 
-export const Settings = (props: RouteComponentProps) => {
+export const Settings = ({
+  flowQueryParams,
+}: { flowQueryParams: QueryParams } & RouteComponentProps) => {
```

Comment:
@millsoper Let me know if I'm missing anything, I think this addresses [your comment here.](https://github.com/mozilla/fxa/pull/14337#discussion_r1005926027)

Reply:
No that's great! I must have misunderstood how the router worked earlier when I was setting it up -- thanks for figuring this out!

Reply:
It's an awkward typing to figure out for sure, I ended up stealing the syntax from `FlowContainer`. 😏